docs/content/development/extensions-core/namespaced-lookup.md

```diff
@@ -46,10 +59,19 @@ described as per the sections on this page. For example:
        "keyColumn": "mykeyColumn",
        "valueColumn": "MyValueColumn",
        "tsColumn": "timeColumn"
-     }
-   ]
+    },
+    "firstCacheTimeout": 120000,
+    "oneToOne":true
+}
  ```
 
+The parameters are as follows
+|Property|Description|Required|Default|
+|--------|-----------|--------|-------|
+|`extractionNamespace`|Specifies how to populate the local cache. See below|Yes|-|
+|`firstCacheTimeout`|How long to wait (in ms) for the first run of the cache to populate. 0 indicates to not wait|No|`60000` (1 minute)|
```

Comment:
what does "first run of the cache" mean? just from reading the documentation it's unclear why one would want to wait, or what the use-case is. Also should would the term "delay" be more appropriate than "timeout" here (similar to `druid.coordinator.startDelay`) ?